Tasks/PowerShellOnTargetMachines2/task.json

```diff
@@ -114,16 +114,35 @@
             "helpMarkDown": "Arguments for the PowerShell script. Can be ordinal parameters or named parameters like -testParam test",
             "groupName": "ScriptOptions"
         },
+        {
+            "name": "InitializationScript",
```

Comment:
Can you check with @RoopeshNair if initialization script should also support inline?